CONTRIBUTING.md

```diff
@@ -0,0 +1,128 @@
+# Contribute Code or Provide Feedback for Azure SDK for .NET
+
+This repository contains a set of Azure SDKs for .NET that can be consumed for Microsoft Azure applications.
+
+## Basics
+
+If you would like to become an active contributor to this project, please follow the instructions provided in [Microsoft Azure Projects Contribution Guidelines](http://azure.github.io/guidelines/).
+
+In the Azure Developer Experience, you are at Step 4:
+
+[API Design Review](https://github.com/Azure/adx-documentation-pr#begin-api-design-review) -> [Engage with ADX team](https://github.com/Azure/adx-documentation-pr/blob/master/README.md#engage-with-adx-team) -> [Swagger specification](https://github.com/Azure/adx-documentation-pr#create-swagger-specification) -> _**[SDKs](https://github.com/Azure/adx-documentation-pr#sdks)**_ -> [CLIs](https://github.com/Azure/adx-documentation-pr#clis)
+
+## Table of Contents
+
+[Before Starting](#before-starting)
+- [Onboarding](#onboarding)
+- [GitHub Basics](#github-basics)
+    - [GitHub Workflow](#github-workflow)
+    - [Forking the Azure/azure-sdk-for-net repository](#forking-the-azureazure-sdk-for-net-repository)
+- [Code of Conduct](#code-of-conduct)
+
+[Filing Issues](#filing-issues)
+
+[Making Changes](#making-changes)
+- [Pull Requests](#pull-requests)
+- [REST API Specs](#rest-api-specs)
+- [Pull Request Guidelines](#pull-request-guidelines)
+    - [Cleaning up commits](#cleaning-up-commits)
+    - [Breaking changes](#breaking-changes)
+    - [General guidelines](#general-guidelines)
+    - [Testing guidelines](#testing-guidelines)
+    - [SDK generation guidelines](#sdk-generation-guidelines)
+
+## Before Starting
+
+### Onboarding
+
+Make sure that your GitHub account is part of the Azure organization. [Use this page](http://aka.ms/azuregithub) to link your account.
```

Comment:
This line is only relevant to internal teams at Microsoft. I think it is better to move it to adx-documentation-pr repo.